Backend/Models/Users/User.cs

```diff
@@ -56,7 +57,7 @@ public bool IsBanned
         private User()
         {
             // This is needed by EntityFramework
-            _socket = new WebSocket("");
+            _socket = new WebSocket("ws://localhost");
```

Comment:
Could we make it nullable since its not mapped anyways?

Reply:
We won't have a user object without a socket though

Reply:
Yeah mainly meant to get rid of this from the constructor

Reply:
We would then need to have multiple null checks for the socket, where logically we wouldn't have a user without a socket